### PR TITLE
Updates for infery 3.9.0

### PR DESCRIPTION
**Pull request by Naveassaf**
- Will merge this once:
   - Its approved
   - Infery 3.9.0 is released

### CODE REVIEW

File: basic/predict_tensorrt.py [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
         # model_path="../../models/hardware_specific_models/nvidia/t4/mobilenet-ssd_t4_optimized.pkl",                       # Nvidia Tesla T4 (TensorRT 8.0.1.6)
         # model_path="../../models/hardware_specific_models/nvidia/v100/mobilenet-ssd_v100_optimized.pkl",                   # Nvidia Volta V100 (TensorRT 8.0.1.6)
         # model_path="../../models/hardware_specific_models/nvidia/jetson/nano/mobilenet-ssd_jetson_nano_optimized.pkl",     # Nvidia Jetson Nano (JetPack 4.6, TensorRT 8.0.1.6)
         # model_path="../../models/hardware_specific_models/nvidia/jetson/xavier/mobilenet-ssd_jetson_xavier_optimized.pkl", # Nvidia Jetson Xavier NX (JetPack 4.6, TensorRT 8.0.1.6)
         # model_path="../../models/hardware_specific_models/nvidia/jetson/orin/mobilenet-ssd_jetson_orin_optimized.pkl",     # Nvidia Jetson Orin AGX (JetPack 5.0, TensorRT 8.4.0.8)
         framework_type="trt",
         logging_verbosity="DEBUG",  # Show TensorRT verbose logs
+        output_device='cpu'         # Copy model output back to CPU. May also be 'gpu'to get a cuda Torch tensor output (no copy)
```
Comment (avideci) [inline basic/predict_tensorrt.py:19]: cool, I would add a 'gpu' output as well to this PR. 
WDYT? 

A snippet that compares both in a single execution.

File: docs/installation.md [hunk 2]
```diff
@@ -2,12 +2,28 @@
 
 ## Environment Requirements
 **infery-cpu:**  
-* Python 3.7, 3.8 or 3.9 (Python 3.9 currently not supported on Windows). 
+* Python 3.7, 3.8 or 3.9 (Windows only supported up to Infery 3.5.0)
 * Other versions of python are supported, but they are not pre-built and compiled for optimization.<br>
 * Please contact us is infery is not supported on your environment.
 
+<br></br>
+**Subpackage Support Matrix:**
```
Comment (avideci) [inline docs/installation.md:10]: Amazing!

File: docs/installation.md [hunk 3]
```diff
@@ -2,12 +2,28 @@
...
+|`openvino_22`|❌|❌|❌|❌|✅|❌|❌|❌|❌|❌|
+|`openvino`|❌|❌|❌|❌|✅|❌|❌|❌|❌|❌|
+|`coreml`|❌|❌|❌|❌|❌|❌|✅|❌|❌|❌|
+
+<br></br>
+
+
 **infery-gpu:**
-  * Python 3.7, 3.8 or 3.9 (Python 3.9 currently not supported on Windows).
+  * Python 3.7, 3.8 or 3.9 (Windows only supported up to Infery 3.5.0)
```
Comment (avideci) [inline docs/installation.md:26]: :fire:

**Comment by Naveassaf** [comment]
Fixed CR feedback :)

**Pull request merged**